### ISSUE TITLE
No way to execute code after a resource has been loaded.

### ISSUE THREAD

**Issue by cat-haines** [pebble/rockyjs#68]
The following snippet demonstrates the problem:

```
(function() {
  var rocky = Rocky.bindCanvas(document.getElementById("pebble"));
  rocky.export_global_c_symbols();
  
  var bmp = rocky.gbitmap_create('http://github.com/apple-touch-icon-57x57.png');
  
  rocky.update_proc = function(ctx, bounds) {
    graphics_draw_bitmap_in_rect(ctx, bmp, [43, 55, 57, 57]);
  };
  
  rocky.mark_dirty();
})();
```

We call `mark_dirty` after setting up our app to initialize the view, however because the image hasn't loaded yet, it's not displayed. Because we're not calling `rocky.mark_dirty()` on a regular basis, the image is never redrawn.

**Comment by HBehrens** [comment]
Does `bmp.onload = function(){rocky.mark_dirty();}` should work. But I think we should finally implement the more user-friendly solution and always call `.mark_dirty()` after a resource was loaded.

**Comment by cat-haines** [comment]
I didn't realize created bitmaps had an onload property - this works great.

I'm not sure we would want to always call `.mark_dirty()` after loading a resource.. you might load a series of resources at app start, and not use them until later.

**Comment by cat-haines** [comment]
Closed and replaced with #69
